depthai_examples/lib/yolov4_spatial_pipeline.cpp

```diff
@@ -0,0 +1,104 @@
+
+#include <depthai_examples/yolov4_spatial_pipeline.hpp>
+#include "depthai/depthai.hpp"
+
+const std::vector<std::string> YoloSpatialDetectionExample::label_map = {"person",         "bicycle",    "car",           "motorbike",     "aeroplane",   "bus",           "train",
+             "truck",          "boat",       "traffic light", "fire hydrant",  "stop sign",   "parking meter", "bench",
+             "bird",           "cat",        "dog",           "horse",         "sheep",       "cow",           "elephant",
+             "bear",           "zebra",      "giraffe",       "backpack",      "umbrella",    "handbag",       "tie",
+             "suitcase",       "frisbee",    "skis",          "snowboard",     "sports ball", "kite",          "baseball bat",
+             "baseball glove", "skateboard", "surfboard",     "tennis racket", "bottle",      "wine glass",    "cup",
+             "fork",           "knife",      "spoon",         "bowl",          "banana",      "apple",         "sandwich",
+             "orange",         "broccoli",   "carrot",        "hot dog",       "pizza",       "donut",         "cake",
+             "chair",          "sofa",       "pottedplant",   "bed",           "diningtable", "toilet",        "tvmonitor",
+             "laptop",         "mouse",      "remote",        "keyboard",      "cell phone",  "microwave",     "oven",
+             "toaster",        "sink",       "refrigerator",  "book",          "clock",       "vase",          "scissors",
+             "teddy bear",     "hair drier", "toothbrush"};
+
+
+void YoloSpatialDetectionExample::initDepthaiDev(std::string nnPath){
+
+    bool syncNN = true;
+    auto colorCam = _p.create<dai::node::ColorCamera>();
+    auto spatialDetectionNetwork = _p.create<dai::node::YoloSpatialDetectionNetwork>();
+    auto monoLeft =  _p.create<dai::node::MonoCamera>();
+    auto monoRight = _p.create<dai::node::MonoCamera>();
+    auto stereo =    _p.create<dai::node::StereoDepth>();
+
+    // create xlink connections
+    auto xoutRgb = _p.create<dai::node::XLinkOut>();
+    auto xoutNN =  _p.create<dai::node::XLinkOut>();
+    auto xoutBoundingBoxDepthMapping = _p.create<dai::node::XLinkOut>();
+    auto xoutDepth = _p.create<dai::node::XLinkOut>();
```

Comment:
Remove `xoutDepth` along with any links to that. Since you are not getting it over the USB the stream gets filled.